src/WebJobs.Script/Description/Binding/DataType.cs

```diff
@@ -10,6 +9,8 @@ public enum DataType
     {
         String,
         Binary,
-        Stream
+        Stream,
+        HttpRequestMessage,
```

Comment:
These http request types don't really belong in this enum. You mentioned this issue to me before I realize - not sure what the right solution is for this, but this isn't ideal